pages/common/octez-client.md

```diff
@@ -0,0 +1,32 @@
+# octez-client
+
+> Interact with the Tezos blockchain.
+> More information: <https://tezos.gitlab.io/user/setup-client.html>.
```

Comment:
A better link would be to <https://tezos.gitlab.io/introduction/howtouse.html#client>, which points to the setup-client page but also explain what the client is.

Reply:
```suggestion
> More information: <https://tezos.gitlab.io/introduction/howtouse.html#client>.
```